packages/code-du-travail-frontend/pages/mentions-legales.js

```diff
@@ -0,0 +1,235 @@
+                Vous pouvez librement désactiver ce service, sur l’ensemble des
+                sites internet susceptibles de l’utiliser en suivant la
+                procédure présentée{" "}
+                <a href="https://www.hotjar.com/legal/compliance/opt-out">
+                  ici: https://www.hotjar.com/legal/compliance/opt-out
+                </a>
+                .
+              </p>
+              <p>
+                Le site utilise également la solution{" "}
+                <a href="https://polyfill.io/v3/">Polyfill.io</a>, ce service
+                permet de faire fonctionner l’ensemble des fonctionnalités du
+                code du travail numérique sur tous les navigateurs, quelless que
+                soient leurs générations.
+              </p>
+              <p>
+                À tout moment, vous pouvez refuser l’utilisation des cookies et
+                désactiver le dépôt sur votre ordinateur en utilisant la
+                fonction dédiée de votre navigateur (fonction disponible
+                notamment sur Microsoft Internet Explorer 11, Google Chrome,
+                Mozilla Firefox, Apple Safari et Opera).
+              </p>
+              <p>
+                L’ensemble de ces informations sont nécessaires à la mise en
+                oeuvre de ce service public numérique.
+              </p>
+              <h1>Accessibilité</h1>
+              <p>
+                La conformité aux normes d’accessibilité numérique est un
+                objectif ultérieur mais nous tâchons de rendre dès la
+                conception, ce site accessible à toutes et à tous.
+              </p>
+              <h2>Signaler un dysfonctionnement</h2>
+              <p>
+                Si vous rencontrez un défaut d’accessibilité vous empêchant
+                d’accéder à un contenu ou une fonctionnalité du site, merci de
+                nous en faire part.
+              </p>
+              <p>
+                Si vous n’obtenez pas de réponse rapide de notre part, vous êtes
+                en droit de faire parvenir vos doléances ou une demande de
+                saisine au Défenseur des droits.
+              </p>
+              <h2>En savoir plus</h2>
+              <p>
+                Pour en savoir plus sur la politique d’accessibilité numérique
+                de l’État :{" "}
+                <a href="http://references.modernisation.gouv.fr/accessibilite-numerique">
+                  http://references.modernisation.gouv.fr/accessibilite-numerique
+                </a>
+              </p>
+              <h1>Sécurité</h1>
+              <p>
+                Le site est protégé par un certificat électronique, matérialisé
+                pour la grande majorité des navigateurs par un cadenas. Cette
+                protection participe à la confidentialité des échanges.
+              </p>
+              <h1>Mentions légales</h1>
+              <p>
+                Le site est édité par l’
```

Comment:
replace  `’`  with `&apos;`

Reply:
👌

Reply:
Ah en fait Thomas a surement raison:
https://english.stackexchange.com/questions/163480/which-is-the-correct-apostrophe-to-use-when-typing-quote-or-backtick
https://www.cs.sfu.ca/~ggbaker/reference/characters/#char2019